### DISCUSSION TITLE
dashboard container startup error

### DISCUSSION THREAD

**Discussion by AndryAsh** [openwisp/docker-openwisp#345] [Q&A] [answered]
There is an error in the log when starting the dashboard container:
attaching to dashboard-1
dashboard-1  | Waiting for database to become available...
dashboard-1  | Connection with database established.
dashboard-1  | Waiting for redis to become available...
dashboard-1  | Connection with redis established.
dashboard-1  | Traceback (most recent call last):
dashboard-1  |   File "manage.py", line 9, in <module>
dashboard-1  |     execute_from_command_line(sys.argv)
dashboard-1  |   File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
dashboard-1  |     utility.execute()
dashboard-1  |   File "/usr/local/lib/python3.7/site-packages/django/core/management/__init__.py", line 345, in execute
dashboard-1  |     settings.INSTALLED_APPS
dashboard-1  |   File "/usr/local/lib/python3.7/site-packages/django/conf/__init__.py", line 83, in __getattr__
dashboard-1  |     self._setup(name)
dashboard-1  |   File "/usr/local/lib/python3.7/site-packages/django/conf/__init__.py", line 70, in _setup
dashboard-1  |     self._wrapped = Settings(settings_module)
dashboard-1  |   File "/usr/local/lib/python3.7/site-packages/django/conf/__init__.py", line 177, in __init__
dashboard-1  |     mod = importlib.import_module(self.SETTINGS_MODULE)
dashboard-1  |   File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
dashboard-1  |     return _bootstrap._gcd_import(name[level:], package, level)
dashboard-1  |   File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
dashboard-1  |   File "<frozen importlib._bootstrap>", line 983, in _find_and_load
dashboard-1  |   File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
dashboard-1  |   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
dashboard-1  |   File "<frozen importlib._bootstrap_external>", line 728, in exec_module
dashboard-1  |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
dashboard-1  |   File "/opt/openwisp/openwisp/settings.py", line 39, in <module>
dashboard-1  |     f'{HTTP_SCHEME}://{os.environ["RADIUS_DOMAIN"]}',
dashboard-1  |   File "/usr/local/lib/python3.7/os.py", line 681, in __getitem__
dashboard-1  |     raise KeyError(key) from None
dashboard-1  | KeyError: 'RADIUS_DOMAIN'
dashboard-1 exited with code 0`
`
Please help to understand.

**Comment by nemesifier** [comment, marked as answer]
It seems you're using and old image. That code mentioning "RADIUS_DOMAIN" was removed.

Can you share the instructions you followed to perform the installation? What image are you using?

**Reply by AndryAsh** [reply]
Hello, I followed the installation instructions posted at https://openwisp.io/docs/dev/docker/user/quickstart.html#using-docker-compose
The images were downloaded with the make start command.
If there are newer images available, please tell me where to download them from.

**Reply by nemesifier** [reply]
You can use the `edge` tag or build them locally with `make develop`.
The images are available from docker hub and/or gitlab container registry

@pandafy we have a problem with `make start`.

**Comment by AndryAsh** [comment]
Don't you have release numbers or container release numbers?

**Comment by pandafy** [comment]
@AndryAsh we are in process of releasing new version of OpenWISP. We are currently working on individual modules and soon will progress with the installers (Ansible and Docker (this repository)). 

You can try the images built on the master branch of the module, by starting the containers with the following command:

```shell
make start TAG=edge
```
